statemachine/src/main/java/com/amazonaws/stepfunctions/cloudformation/statemachine/ResourceHandler.java

```diff
@@ -85,7 +85,7 @@ protected void verifyStateMachineArnIsPresent(String resourceArn) throws AmazonS
         }
     }
 
-    protected AmazonServiceException getStateMachineDoesNotExistException() {
+    private AmazonServiceException getStateMachineDoesNotExistException() {
```

Comment:
suggestion (non-blocking): if this was the only usage site, given the size of this method it's probably cleaner to just inline it. I *think* the only usage site is in `verifyStateMachineArnIsPresent`.

I don't think this exception *needs* to be abstracted into its own method anymore as it doesn't really do/provide much.

Reply:
+1 not blocking from my side, only nit